eav/models/entity.py

```diff
@@ -200,6 +207,50 @@ def __iter__(self):
         """
         return iter(self.get_values())
 
+    def save_bulk(
+        self,
+        eav_values,
+        attributes,
+    ):
```

Comment:
In line with the other comment, should `bulk_save()` take a `list[Entity]`? Or is the use case for this something different than the two scenarios I identified earlier?

Thinking of how Django's `bulks_create()` works, you pass in a list of objects created in memory that then get written with an optimized database query. In this scenario, I would think that when wanting to update a number of entities, you would do something in your app like:

```
for patient in patients:
  patient.eav.age = 10
  patient.eav.height = 2.3
  
eav.bulk_save(patients, 10)
Patient.objects.bulk_create(patients)
```

Note I did a generic `eav.bulk_save()` as I'm not sure this should be on the Entity manager as inserted into a specific Entity instance versus being on a Model Manager. Maybe that's a future thing, having a manager for a model that has EAV methods on it, but for now, I think a utility function would work just fine too.

Also, thinking about it, we should use the same language as Django with "bulk_update" and their params to not have confusion around things like expecting signals to be called like a typical `save()`

What do you think?